diff -Naur ros-humble-ros-workspace-1.0.2_org/CMakeLists.txt ros-humble-ros-workspace-1.0.2/CMakeLists.txt
--- ros-humble-ros-workspace-1.0.2_org/CMakeLists.txt	2023-05-13 09:39:51.509756650 +0800
+++ ros-humble-ros-workspace-1.0.2/CMakeLists.txt	2023-05-13 09:40:24.537975002 +0800
@@ -14,7 +14,7 @@
   set(PYTHON_INSTALL_DIR "lib/python${PYTHON_MAJOR_MINOR}/site-packages")
   set(SHELL_EXT "sh")
 endif()
-set(AMENT_PACKAGE_DIR "${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_DIR}/ament_package")
+set(AMENT_PACKAGE_DIR "$ENV{STAGING_DIR_NATIVE}/usr/${PYTHON_INSTALL_DIR}/ament_package")
 if(NOT EXISTS "${AMENT_PACKAGE_DIR}")
   # Check for an .egg-link file and use the listed directory if it exists
   get_filename_component(AMENT_PACKAGE_EGG_LINK "${AMENT_PACKAGE_DIR}" DIRECTORY)
