diff -Naur ros-humble-rsl-0.2.1_org/tests/CMakeLists.txt ros-humble-rsl-0.2.1/tests/CMakeLists.txt
--- ros-humble-rsl-0.2.1_org/tests/CMakeLists.txt	2023-05-07 11:53:18.215360070 +0800
+++ ros-humble-rsl-0.2.1/tests/CMakeLists.txt	2023-05-07 11:59:24.986324119 +0800
@@ -1,7 +1,8 @@
-list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
-
-find_package(Catch2 3.2.0 REQUIRED)
 find_package(range-v3 REQUIRED)
+find_package(Catch2 3.2.0 REQUIRED)
+if (NOT Catch2_FOUND)
+	list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
+endif()
 
 add_executable(test-rsl
     algorithm.cpp
@@ -19,4 +20,5 @@
     Catch2::Catch2WithMain
     range-v3::range-v3
 )
+include(Catch)
 catch_discover_tests(test-rsl)
