diff -Naur ros-humble-console-bridge-vendor-1.4.1_org/CMakeLists.txt ros-humble-console-bridge-vendor-1.4.1/CMakeLists.txt
--- ros-humble-console-bridge-vendor-1.4.1_org/CMakeLists.txt	2023-04-26 16:01:55.653557871 +0800
+++ ros-humble-console-bridge-vendor-1.4.1/CMakeLists.txt	2023-04-26 16:03:50.311237487 +0800
@@ -71,7 +71,7 @@
 
   # Download and build console_bridge
   externalproject_add(console_bridge-1.0.2
-    URL https://github.com/ros/console_bridge/archive/1.0.2.tar.gz
+    URL ${CMAKE_CURRENT_SOURCE_DIR}/1.0.2.tar.gz
     URL_HASH SHA256=303a619c01a9e14a3c82eb9762b8a428ef5311a6d46353872ab9a904358be4a4
     TIMEOUT 600
     CMAKE_ARGS
